### PR TITLE
Add typography max width

### PR DESCRIPTION
**Pull request by marciopmoreira6**
Change small font size (12px) from regular to medium

### CODE REVIEW

File: design-tokens/tokens/layer1/base.json [hunk 1]
```diff
@@ -428,10 +428,10 @@
           }
         },
         "large": {
-          "regular": {
+          "medium": {
             "value": {
               "fontFamily": "{echoes.fontFamily.sans}",
-              "fontWeight": "{echoes.fontWeight.regular}",
+              "fontWeight": "{echoes.fontWeight.medium}",
               "lineHeight": "{echoes.lineHeight.30}",
```
Comment (jeremy-davis-sonarsource) [inline design-tokens/tokens/layer1/base.json:435]: Why is `regular` disappearing from `large` ?

File: design-tokens/tokens/layer1/base.json [hunk 2]
```diff
@@ -380,21 +380,21 @@
...
             },
             "type": "typography"
           },
-          "semi-bold": {
+          "bold": {
             "value": {
               "fontFamily": "{echoes.fontFamily.sans}",
-              "fontWeight": "{echoes.fontWeight.semi-bold}",
+              "fontWeight": "{echoes.fontWeight.medium}",
               "lineHeight": "{echoes.lineHeight.20}",
```
Comment (jeremy-davis-sonarsource) [inline design-tokens/tokens/layer1/base.json:398]: Why is `medium` called `bold` for default size, but `medium` for `large`?

**Review by jeremy-davis-sonarsource — CHANGES_REQUESTED** [review]
I'm confused :upside_down_face:

File: design-tokens/tokens/layer1/base.json (hunk 1 above)
Comment (marciopmoreira6) [reply design-tokens/tokens/layer1/base.json]: 🤦‍♂️ 

It should be fixed now

File: design-tokens/tokens/layer1/base.json (hunk 2 above)
Comment (marciopmoreira6) [reply design-tokens/tokens/layer1/base.json]: It should be fixed now too

File: src/generated/design-tokens-base.css [hunk 3]
```diff
@@ -104,11 +104,11 @@
   --echoes-text-decoration-none: none;
   --echoes-text-decoration-underline: underline;
   --echoes-typography-text-default-regular: 400 0.875rem/1.25rem Inter;
-  --echoes-typography-text-default-medium: 500 0.875rem/1.25rem Inter;
   --echoes-typography-text-default-semi-bold: 600 0.875rem/1.25rem Inter;
+  --echoes-typography-text-default-bold: 500 0.875rem/1.25rem Inter;
   --echoes-typography-text-small-regular: 400 0.75rem/1.125rem Inter;
   --echoes-typography-text-small-semi-bold: 600 0.75rem/1.125rem Inter;
```
Comment (jeremy-davis-sonarsource) [inline src/generated/design-tokens-base.css:110]: still no `medium` for `small` text :sweat_smile:

**Pull request merged**